packages/terra-application-name/tests/wdio/application-header-name-spec.js

```diff
@@ -6,7 +6,7 @@ describe('ApplicationHeaderName', () => {
   describe('Displays a default application header name', () => {
     beforeEach(() => browser.url('/#/raw/tests/application-name/application-header-name-default'));
 
-    Terra.should.matchScreenshot({ viewports, options: { selector: '#default' } });
+    Terra.should.matchScreenshot(viewports, { selector: '#default' });
```

Comment:
To correctly create the screenshots for each viewport, you write matchScreenshot like this:
`Terra.should.matchScreenshot({viewports, selector: '#default' });`

If you noticed, most of your VR screenshots were removed.

Reply:
Oops, updated